Description: Handle _error becoming a smart pointer thingy
 This should be fixed nicer in apt, so that we can use _error again somehow, as
 _GetErrorObj() is not really part of the API (but the ABI).
Author: Julian Andres Klode <juliank@ubuntu.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/main.cc
+++ b/src/main.cc
@@ -1154,7 +1154,7 @@ int main(int argc, char *argv[])
 	{
 	  // Connect up the "please consume errors" routine for the
 	  // command-line.
-	  consume_errors.connect(sigc::mem_fun(_error, (void (GlobalError::*)()) &GlobalError::DumpErrors));
+	  consume_errors.connect(sigc::mem_fun(_GetErrorObj(), (void (GlobalError::*)()) &GlobalError::DumpErrors));
 
 	  if(update_only || install_only || autoclean_only || clean_only)
 	    {
